Sources/StreamChat/Models/ChannelPinningScope.swift

```diff
@@ -0,0 +1,11 @@
+//
+// Copyright © 2024 Stream.io Inc. All rights reserved.
+//
+
+import Foundation
+
+/// The scope of the channel pinning action.
+public enum ChannelPinningScope: String {
+    /// Channel is pinned only for the currently connected user. Other channel members do not see channels as pinned.
+    case me
+}
```

Comment:
Since this not meant to be used by the customer to perform switching it should be fine. But in theory, adding a new case would be a breaking change. But I guess it is fine in this case 👍